src/pytorch_lightning/plugins/precision/apex_amp.py

```diff
@@ -43,7 +42,7 @@ def __init__(self, amp_level: str = "O2") -> None:
         self._connected = False
         self._state_dict_loaded = False
 
-    def main_params(self, optimizer: Optimizer) -> _PARAMETERS:
+    def get_main_params(self, optimizer: Optimizer) -> _PARAMETERS:
```

Comment:
I should probably revert his api change in Lite so we don't have to deprecate/rename it